### ISSUE TITLE
Foundry summons doesnt work with remaster or summon spells

### ISSUE THREAD

**Issue by GuitarGuyNick** [MrVauxs/pf2e-jb2a-macros#176]
foundry summons needs fixing as to how it interacts with summon spells. 

try using summon monitor. or summon undead.

**Comment by MrVauxs** [comment]
I would appreciate more info on what is wrong.

**Comment by GuitarGuyNick** [comment]
> I would appreciate more info on what is wrong.

the foundry summons doesnt trigger and provide the summons menu. when they cast it it doesnt trigger the module.

**Comment by MrVauxs** [comment]
Nope, it sure works. Go to PF2e Animation Macros settings, press the "update menu" button, and accept the changes as it looks like you didn't accept it when the module updated.

Uploading chrome_4Xnrum7E4b.mp4…

**Comment by GuitarGuyNick** [comment]
could you try with summon monitor? thats what made me make this request. and when we were in session on saturday it didnt work.

**Comment by MrVauxs** [comment]
Summon Monitor does not exist indeed. Will add it.
